MdeModulePkg/Universal/Variable/RuntimeDxe/VariablePolicyLockingCommon.c

```diff
@@ -61,6 +62,11 @@ LockPolicyInterfaceAtReadyToBoot (
 {
   EFI_STATUS  Status;
 
+  if ((GetDeviceState () & DEVICE_STATE_UNIT_TEST_MODE) != 0) {
+    DEBUG ((DEBUG_INFO, "[%a] Unit test mode is enabled. Skipping lock.\n", __FUNCTION__));

```

Comment:
I would be tempted to put this at DEBUG_ERROR level, because the only time you would be using this is running unit tests, in which case, great, whatever, another print, but if you happened to his this in prod code, that would be a very big error.

I know there has been discussion in this I have not been involved with, but should this be enclosed in DEBUG_CODE so that we don't have any possibility of doing this on a release build?

Reply:
I think `DEBUG_CODE` would be a good addition.

Reply:
This call is blocked behind DEBUG_CODE. The print level is unchanged